src/reactGUI/DOM-shim.coffee

```diff
@@ -0,0 +1,17 @@
+try
+  DOM = require 'react-dom-factories'

```

Comment:
Same here (and in the filename): please could this be `ReactDOM = ...` and `ReactDOM-shim.coffee`?

Reply:
There are actually three packages now which all have different content, `react`, `react-dom` and `react-dom-factories`, that export globals `React`, `ReactDOM` and `ReactDOMFactories` respectively. The first two already have shims, so `ReactDOM-shim.coffee` already exists, but it doesn't include the factories.

I can change the file name of the new shim in this commit to match the global name (rather than the property name that it is usually referred to by) to make it clearer which package is being shimmed, but to keep the code that uses it clear go on using `DOM`, since `ReactDOMFactories` is a little verbose.